src/mach.rs

```diff
@@ -750,6 +800,12 @@ impl<'a> Mach<'a> {
         //////////////////////////////
         for data in self.data {
             file.write_all(data.data)?;
+
+            if let Some(&align_pad) = self.segment.align_pad_map.get(data.name) {
+                for _ in 0..align_pad {
+                    file.write_all(&[0xaa])?;
```

Comment:
Why `0xaa`

Reply:
No particular reason. Just didnt want to use `0x00`, as it is confusable for a zero pointer when you accidentially read past the end of a data object.

Reply:
This should be commented because it’s a magic number, and future reader will wonder the same

Reply:
Done